Backend/EzPay/src/main/java/com/ezpay/config/ApplicationConfig.java

```diff
@@ -47,6 +47,4 @@ public UserDetailsService userDetailService() {
                 .orElseThrow(() -> new UsernameNotFoundException("User not found with email: " + email));
```

Comment:
Existe un constructor sobrecargado para recibir una String en la clase UsernameNotFoundException que recibe el email y muestra directamente el mensaje, es seconstructor el que estas usando verdad?, entonces solo deberias enviarle el email sin el texto de "User not found with email"